### PR TITLE
chore(channels): promote alpha to stable

### PR DESCRIPTION
**Pull request by moshevayner**
**What this PR does / why we need it**:
Promotes alpha channel to stable channel with changes from #16284

**Special notes for your reviewer**:

### CODE REVIEW

File: channels/stable [hunk 1]
```diff
@@ -103,137 +103,137 @@ spec:
     networking:
       kubenet: {}
   kubernetesVersions:
-  - range: ">=1.28.0"
-    recommendedVersion: 1.28.5
```
Comment (hakman) [inline channels/stable:107]: @moshevayner Could revert the indentation change?

Comment (moshevayner) [reply channels/stable:107]: Ooof, VSCode on a new computer 🤦🏼‍♂️ 
Sure, let me take care of that.

**Review by hakman — APPROVED** [review]
Thanks!

**Pull request merged**